ios/Classes/ComAppceleratorUrlSessionSessionProxy.m

```diff
@@ -25,6 +25,15 @@ - (id)_initWithPageContext:(id<TiEvaluator>)context andArguments:(NSDictionary*)
     
     return self;
 }
+-(void)_initWithProperties:(NSDictionary *)properties
+{
+    sessionConfiguration = [properties objectForKey:@"configuration"];
+    [self rememberProxy:sessionConfiguration];
+    session = [NSURLSession sessionWithConfiguration:[sessionConfiguration sessionConfiguration]
+                                            delegate:(id<NSURLSessionDelegate>)[[UIApplication sharedApplication] delegate]
+                                       delegateQueue:nil];
+    [super _initWithProperties:properties];
```

Comment:
It was needed because the `_initWithPageContext:andArguments` is not being called when you are instantiating the object directly from JavaScript, which was leaving the session instance variable unset. I left `_initWithPageContext:andArguments` to handle the legacy case, when we are initialising from inside the legacy module method. I am not sure if there is another way of doing this, as the properties are handled once the object is already initialised, afaik.